npm-cli/settings.yml

```diff
@@ -1,5 +1,11 @@
 # These are the settings that npm-cli repos should inherit, it contains the base settings they should have but can extend if needed
 
+branches:
+  - name: *
```

Comment:
All branches require a PR?

Reply:
All PRs on any branch require an approval.  Reminder this won't affect the registry team this is only going to apply to the npm cli repos.